build-system/test-configs/forbidden-terms.js

```diff
@@ -103,6 +103,71 @@ const forbiddenTermsGlobal = {
   'describes.*\\.only': {
     message: 'Please remove all instances of describes.only',
   },
+  'this.timeout': {
+    message: 'Tests are not allowed to override the default timeout',
+    allowlist: [
+      'ads/google/a4a/test/test-utils.js',
```

Comment:
Two requests:
1. For maintainability, can we separate the infrastructure files, actual test case files, and other files like `.md` files, `third_party` files, etc. into three separate sub-lists within the same allowlist?
2. Can we then add a TODO with a link to your tracking issue above the sub-list of test case files so that we can keep tabs on it?